pkg/pattern/aggregation/push.go

```diff
@@ -142,6 +147,7 @@ func NewPush(
 		entries: entries{
 			entries: make([]entry, 0),
 		},
+		metrics: NewMetrics(registrer, "pattern_ingester"),
```

Comment:
should the namespace be `loki` here? we already have the subsystem as `pattern_ingester`, so I think this would make the metric have `pattern_ingester` in it twice?

Reply:
yes, missed this one. Fixed.